crates/red_knot_workspace/src/workspace.rs

```diff
@@ -143,9 +143,7 @@ impl Workspace {
             new_packages.insert(path, package);
         }
 
-        self.set_package_tree(db)
-            .with_durability(Durability::MEDIUM)
```

Comment:
We no longer need to specify the durability because of this salsa change: https://github.com/salsa-rs/salsa/pull/559